### PR TITLE
Moved onto upstream CHERIoT RTOS and reorganised the repository

### PR DESCRIPTION
**Pull request by HU90m**
All examples now live in an example repository. They will be moved out of `examples/all` with time.

### CODE REVIEW

File: compartments/lcd_test.cc [hunk 1]
```diff
@@ -12,15 +12,11 @@ void __cheri_compartment("lcd_test") lcd_test()
 {
 	using namespace sonata::lcd;
 
-	auto lcd      = SonataLCD();
+	auto lcd      = SonataLcd();
```
Comment (AlexJones0) [inline compartments/lcd_test.cc:15]: I like LCD

File: .gitmodules [hunk 2]
```diff
@@ -6,6 +6,6 @@
 	url = https://github.com/lowrisc/cheriot-rtos
 	branch = sonata
 [submodule "display_drivers"]
-	path = display_drivers
+	path = third_party/display_drivers
```
Comment (nbdd0121) [inline .gitmodules:9]: This change should ideally go in a separate commit

**Review by nbdd0121 — APPROVED** [review]

**Comment by HU90m** [comment]
@nbdd0121 any idea why CI is blocked?

**Comment by GregAC** [comment]
> All examples now live in an example repository.

Do you mean directory here or is the intent to start a new repository for examples? If so for what reason? One of the original purposes of this repository was to contain examples and I'd rather avoid another repository and more sub-modules.

**Comment by HU90m** [comment]
> Do you mean directory here or is the intent to start a new repository for examples?

Ah yes, sorry, I meant directory
